src/RoslynPad.Common/Runtime/ResultObject.cs

```diff
@@ -99,7 +101,17 @@ private void Initialize()
 
             if (_property != null)
             {
-                var value = _property.GetValue(_o);
+                object value;
+                try
+                {
+                    value = _property.GetValue(_o);
+                }
+                catch (TargetInvocationException exception)
+                {
+                     _header = $"{_property.Name} = Threw {exception.InnerException.GetType().Name}";
```

Comment:
In this case, you can also add the exception object as a child so it could be fully inspected.